grpc/requests_common.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <thread>
+#include <utility>
+#include <memory>
+
+#include <blackhole/attribute.hpp>
+
+#include <grpcpp/grpcpp.h>
+#include <grpcpp/server.h>
+#include <grpcpp/server_builder.h>
+#include <grpcpp/server_context.h>
+
+#include "elliptics.grpc.fb.h"
+
+#include "elliptics/utils.hpp"
+
+#include "library/elliptics.h"
+#include "library/logger.hpp"
+#include "library/n2_protocol.hpp"
+
+namespace fb = flatbuffers;
+namespace ell = ioremap::elliptics;
```

Comment:
Please, avoid namespace aliases in headers.

Reply:
Also avoid alias for local namespaces.

Reply:
> local namespaces

Did you mean project namespaces, or namespaces that are upgraded in current source file?